### ISSUE TITLE
Calendar: does not close with ESC key in scan mode

### ISSUE THREAD

**Issue by dollykumari5** [OfficeDev/office-ui-fabric-react#9713]
Detailed description of this issue can be found here [Sharepoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/ES972Zd6XgFOvx30t-qGtCEBHpiqyKiMF6t2Hi002BSjvg?e=VBwDJb)
Component: Calendar

**Comment by khmakoto** [comment]
Hi @dollykumari5, thanks for filing this issue! This seems to be a duplicate of Bug 12 in #8349, so I'll be closing this since the issue is already being tracked there. Thanks again!

**Comment by dollykumari5** [comment]
Hi @khmakoto we will be closing the parent bug [#8349](https://github.com/OfficeDev/office-ui-fabric-react/issues/8349) and logging individual bug in GitHub as advised by @micahgodbolt.

@awalia13 for tracking.

**Comment by micahgodbolt** [comment]
Last time we chatted, I thought the tagging limitation of GitHub meant that you were going to be logging these bugs internally for better tracking. Is that not still the case?

**Comment by micahgodbolt** [comment]
Also, if we do log the bug here in github, we need to have a much better description in github itself. bug description, repro steps etc. That should all be public as the repro should be on our own site.

**Comment by dzearing** [comment]
This may have already been resolved, but epic huge a11y pass bugs are being closed, as are internal VSTS. We are tracking with individual issues, and I'll open this one to ensure it's fixed as it is not yet listed as resolved in the pass bug.

**Comment by lorejoh12** [comment]
tested on latest Edge with latest OUFR Calendar at https://fabricweb.z5.web.core.windows.net/oufr/7.59.3/#/examples/datepicker. This issue seems to be resolved. Closing.
